doc/PRE_UPGRADE.md

```diff
@@ -1,3 +1,4 @@
 If you are upgrading to a new major version of Nextcloud, please make sure that your Nextcloud apps are up to date from Nextcloud's administration panel beforehand.
 
 Additionally, if you installed specific Nextcloud apps, we recommend making sure that they are compatible with the new major version. YunoHost will attempt to check this automatically at the very beginning of the upgrade, but a manual check doesn't hurt either. For Nextcloud 28, this forum thread might be helpful : <https://help.nextcloud.com/t/apps-not-compatible-with-nc-28/176234>.
+
```

Comment:
```suggestion
```

Unecessary new line ... Honestly at this point the history should be rewritten or squashed, half the commits were cancelled by other commits ...

Reply:
(final empty lines in md files are added by some markdown formaters, because it seems to be standardized like thant)